app/controllers/admin/enterprise_fees_controller.rb

```diff
@@ -10,7 +10,7 @@ def index
 
       blank_enterprise_fee = EnterpriseFee.new
       blank_enterprise_fee.enterprise = current_enterprise
-      3.times { @collection << blank_enterprise_fee }
+      3.times { @collection.to_a << blank_enterprise_fee }
```

Comment:
3 times?? :thinking: I don't get it.

Reply:
This is existing functionality on the enterprise fees page, we add three blank fees to the UI when the page loads so they are ready to be filled out :shrug: